currency-map.js

```diff
@@ -362,5 +362,337 @@ module.exports = {
     "TWD": {
         "name": "Taiwan Dollar (TWD)",
         "symbolFormat": "TWD {#}"
+    },
+    "CZK": {
+        "name": "Czech Koruna",
+        "symbolFormat": "CZK {#}"
+    },
+    "FIM": {
+        "name": "",

```

Comment:
Those 4 blank name currencies are all old - and have been replaced by the Euro. They should not be selectable from the survey.